chain/gen/genesis/miners_binding.go

```diff
@@ -0,0 +1,105 @@
+package genesis
+
+import (
+	"bytes"
+	"context"
+	"crypto/rand"
+	"encoding/json"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/lotus/chain/types"
+	"github.com/filecoin-project/lotus/chain/vm"
+	"github.com/filecoin-project/lotus/genesis"
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/filecoin-project/specs-actors/actors/builtin"
+	init_ "github.com/filecoin-project/specs-actors/actors/builtin/init"
+	"github.com/filecoin-project/specs-actors/actors/builtin/power"
+	cbor "github.com/ipfs/go-ipld-cbor"
+	ic "github.com/libp2p/go-libp2p-core/crypto"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"golang.org/x/xerrors"
+)
+
+func GetAccountMeta(account genesis.Actor) (*genesis.AccountMeta, error) {
+	var meta genesis.AccountMeta
+	meatb, err := account.Meta.MarshalJSON()
+	if err != nil {
+		return nil, err
+	}
+	err = json.Unmarshal(meatb, &meta)
```

Comment:
`account.Meta` is `json.RawMessage`, This could just be

```suggestion
	err = json.Unmarshal(account.Meta, &meta)
```